src/vm/amd64/unixasmhelpers.S

```diff
@@ -172,6 +172,98 @@ NESTED_ENTRY JIT_RareDisableHelper, _TEXT, NoHandler
 
 NESTED_END JIT_RareDisableHelper, _TEXT
 
+#ifdef FEATURE_HIJACK
+
+//------------------------------------------------
+// OnHijackScalarTripThread
+//
+NESTED_ENTRY OnHijackScalarTripThread, _TEXT, NoHandler
```

Comment:
Done in my latest commit, but I now need to update HijackArgs to include rdx. Will update it soon.